packages/divider-curve/src/assets/divider-curve-variation-two.tsx

```diff
@@ -0,0 +1,18 @@
+const DividerCurveTwo = () => (
+  <svg
+    role="separator"
+    viewBox="0 0 1921 27"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    preserveAspectRatio="none"
+  >
+    <title>Curved Dividing Line</title>
+    <desc>A curved line dividing sections.</desc>
+    <path
+      d="M0.189453 4.69347C79.7719 11.8196 135.64 11.4505 222.462 9.58349C925.957 -5.54451 1280.29 -0.845186 1750.02 23.8048C1797.52 26.2977 1871.29 26.2977 1920.19 23.8048"
+      stroke="#131415"
+    />
```

Comment:
Since the only difference is this line and `viewBox`, can't we use a prop to render the difference?

Reply:
Using `inverted` prop to achieve this